webapp/client/src/pages/LogoutPage.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import FormHeader from "../components/FormHeader";
+import StepHeaderButtons from "../components/StepHeaderButtons";
+import StepForm from "../components/StepForm";
+
+export default function LogoutPage({ form }) {
+  const { t } = useTranslation();
+  const stepHeader = {
+    title: t("form.logout.title"),
+    intro: t("form.logout.intro"),
+  };
+  return (
+    <>
+      <StepHeaderButtons />
+      <FormHeader {...stepHeader} />
+      <StepForm {...form} nextButtonLabel={t("form.logout.button")} />
+    </>
+  );
+}
+
+LogoutPage.propTypes = {
+  form: PropTypes.exact({
+    action: PropTypes.string,
+    csrfToken: PropTypes.string,
+    showOverviewButton: PropTypes.bool,
```

Comment:
We will never show the OverviewButton on this page, so we could just omit it and set it directly to false in line 18.

Reply:
You mean like this: 
![image](https://user-images.githubusercontent.com/97096480/149964370-7405fb36-1bcf-4def-86b7-b7f3ca7fbcfb.png)
Otherwise an warning is thrown: Warning: Failed prop type: Invalid prop `form` key `showOverviewButton` supplied to `LogoutPage`.

Reply:
We talked about it and the solution was to remove the showOverviewButton prop